xml/deployment_installation.xml

```diff
@@ -825,9 +825,253 @@
       configured separately and differently from the &worker_node;s.
      </para>
     </sect2>
+
+    <sect2 xml:id="sec.deploy.install.qcow2.procedure.openstack">
+     <title>Installation on &soc;</title>
+     <para>
+      You can easily deploy a &productname; on &soc; using &ostack;. 
+      You will need a &productname; machine image and &ostack; Heat templates.
+      Once you have created a stack you will continue with the &productname; setup.
+     </para>
+     <note>
+      <title>&productname; machine image for &soc;</title>
+      <para>
+       Download the latest SUSE CaaS Platform for OpenStack image from 
```

Comment:
use the &productname; and &ostack entities.

Reply:
Good catch